api/auth.js

```
@@ -24,11 +24,17 @@ import crypto  from 'crypto';
 const router = express.Router();
 
-// ─── Secret resolution (FIX: no throw on missing env var) ────────────────────
-if (!process.env.LICENSE_JWT_SECRET) {
-  throw new Error('LICENSE_JWT_SECRET environment variable is required');
-}
-const LICENSE_SECRET = process.env.LICENSE_JWT_SECRET;
+// ─── Secret resolution ───────────────────────────────────────────────────────
+// In production the env var is required. In dev/test/preview fall back to a
+// deterministic placeholder so the API module doesn't crash-start.
+const LICENSE_SECRET = (() => {
+  if (process.env.LICENSE_JWT_SECRET) return process.env.LICENSE_JWT_SECRET;
```

ESLint

[error] 'process' is not defined.

(no-undef)